draft-demarco-oauth-status-attestations.md

```diff
@@ -273,6 +273,8 @@ the public key contained in the Digital Credential;
 The technical and details about the `credential_pop` object
 are defined in the next section.
 
+*TODO CREDENTIAL_POP DEFINITION*
```

Comment:
missing definition here, please provide the definition of credential_pop, type Array[String] and the description of this parameter, before the validation requirements at line 266